assets/app/view/game/buy_trains.rb

```diff
@@ -281,6 +281,8 @@ def render
           children.concat(render_president_contributions)
         end
 
+        children << h(:div, @step.ebuy_custom_message(@corporation)) if @step.respond_to?(:ebuy_custom_message)
```

Comment:
I'm not a fan of half solutions. If we need to move ebuy text into steps (which I do think is a good idea), let's move the default text into the train module and call it ebuy_message.